### ISSUE TITLE
TRS spec WD header is not being rendered

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/oslc-specs#28]

**Comment by berezovskyi** [comment]
@ndjc how do go about debugging what's the matter and how to fix it?

**Comment by berezovskyi** [comment]
we discussed this with Jim and agreed we need you help, Nick!

![image](https://user-images.githubusercontent.com/64734/57469739-9eb00d00-7287-11e9-943c-bae523af8eb9.png)

**Comment by ndjc** [comment]
Fixed in the latest 2.0.3 version of respec.
I use Safari, and the Javascript console there is pretty good for debugging respec. In this case, the console showed an error that conf.thisVersion.replace was failing because thisVersion was undefined, so I searched the source for that call, and found that indeed there was a missing assignment of a default empty string in case that variable was not defined in the spec's config section.
